features/prefers-contrast.yml

```
@@ -0,0 +1,6 @@
+name: prefers-contrast media query
+description: The `prefers-contrast` CSS media query sets styles based on whether the user prefers more or less contrast.
```

A little circular. The minimalist rewrite:

```suggestion
description: The `prefers-contrast` CSS media query sets styles based on whether the user prefers more or less color contrast.
```

And the maximalist one:

```suggestion
description: The `prefers-contrast` CSS media query sets styles based on whether the user prefers more or less contrast, the difference between foreground and background colors.
```

I'm not sure which I like better, so you can decide. 😄